services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -232,14 +222,29 @@ def getReportSpecificEmailsByReportType(self):
             if notifiy_email not in unique_recipients:
                 unique_recipients.add(notifiy_email)
 
-            if ntf[1] and PERM_RECL_EMAIL not in unique_recipients:
+            if ntf[1] and self.mine.major_mine_ind and PERM_RECL_EMAIL not in unique_recipients:
                 unique_recipients.add(PERM_RECL_EMAIL)
 
-            if ntf[2] and regional_email not in unique_recipients:
+            if ntf[2] and not self.mine.major_mine_ind and regional_email not in unique_recipients:
                 unique_recipients.add(regional_email)
 
         return list(unique_recipients)
 
+    def send_report_update_email(self, is_edit):
```

Comment:
would maybe rename this to send_crr_report_update_email for clarity

Reply:
This was an existing method. And this is being used in `..resources/mine_report.py`

Reply:
Well, it was an exiting function, i think as name says its for report update. This has has been calling from ../resource/mine_report.py.

To make things clearer, this pr is for addition of crr or prr. So I didn't want to change an existing name to confuse things and separate logics inside with another if. Instead introduce this new function with a clearer name.

I think this way is better than putting every thing in to one function and separate things inside using another if.

Reply:
This function will break if a PRR report is passed to it and so it is specifically for CRRs. I think it's better to change the name of the function. Having an inaccurate name that does not reflect what the function does is confusing.

Reply:
@taraepp i think its easy to get into a call and discuss, waiting in discord.

`send_report_update_email` function is there from 1.5 tag (2021). And its been using in other places, which is not part of this ticket (5658). And also I used the same method thinking the ticket is only about CRR initially. Then after clarifications PRR was included as well, hence I introduced this separate method `send_crr_and_prr_add_notification_email` to make things clearer.
But if you think both these needs to merge, I suggest to do that in a separate ticket as `..resources/mine_reports.post()` is not part of this ticket, which needs to be tested if we merge these.